src/PrestaShopBundle/Resources/config/services/core/grid/grid_factory.yml

```diff
@@ -231,10 +231,35 @@ services:
             - '@prestashop.core.grid.filter.form_factory'
             - '@prestashop.core.hook.dispatcher'
 
+
     prestashop.core.grid.factory.tax_rules_group:
       class: 'PrestaShop\PrestaShop\Core\Grid\GridFactory'
       arguments:
         - '@prestashop.core.grid.definition.factory.tax_rules_group'
         - '@prestashop.core.grid.data.factory.tax_rules_group'
         - '@prestashop.core.grid.filter.form_factory'
         - '@prestashop.core.hook.dispatcher'
+        
+    prestashop.core.grid.grid_factory.order_message:
+      class: 'PrestaShop\PrestaShop\Core\Grid\GridFactory'
+      arguments:
+        - '@prestashop.core.grid.definition.factory.order_message'
+        - '@prestashop.core.grid.data.factory.order_message'
+        - '@prestashop.core.grid.filter.form_factory'
+        - '@prestashop.core.hook.dispatcher'
+
+    prestashop.core.grid.factory.attachment:
+      class: 'PrestaShop\PrestaShop\Core\Grid\GridFactory'
+      arguments:
+        - '@prestashop.core.grid.definition.factory.attachment'
+        - '@prestashop.core.grid.data_factory.attachment_decorator'
+        - '@prestashop.core.grid.filter.form_factory'
+        - '@prestashop.core.hook.dispatcher'
+
+    prestashop.core.grid.factory.attachment:
```

Comment:
Damned, it seems I created duplicates 😭

Reply:
I guess it will be my mistake from file list page rebase but i am not sure how to approach this? Should there be a separate PR or can it be fixed in this?